packages/store/test/StoreCoreDynamicUpdate.t.sol

```diff
@@ -0,0 +1,142 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { Test, console } from "forge-std/Test.sol";
```

Comment:
do we want to update everything to use explicitly specified imports? (console is needed because gas-report doesn't seem to autoinject it)

Reply:
Personally I think using file imports for `Test` and `Script` is fine, but no strong opinion.

For reference - [Foundry best practices](https://book.getfoundry.sh/tutorials/best-practices)
<img width="828" alt="CleanShot 2023-04-21 at 19 07 34@2x" src="https://user-images.githubusercontent.com/89248902/233694965-dec7f6c7-666f-4c1c-bda6-3065a4b8163a.png">